packages/website/src/components/Navbar/Navbar.tsx

```diff
@@ -24,12 +25,12 @@ export const Navbar = () => {
         position="relative"
       >
         {baseNavbar}
-        <Flex>
-          <Button size={buttonSize} mr="12px" onClick={() => openModal('mentor')}>
+        <Flex overflow="hidden">
```

Comment:
❓ Why do we need overflow to be hidden here?

Reply:
It seems to me that with overflow hidden, larger buttons do not destroy the page layout. This can be seen especially in the curriculum section of the iPhone 6 mobile view.

Reply:
But I can see that after deployment, the page looks different than for me locally.